components/Snackbar/tests/unit/MDCSnackbarMessageViewTests.m

```diff
@@ -91,7 +91,8 @@ - (void)testAccessibilityLabelSetFromSnackbarMessageProperty {
                         self.message.accessibilityLabel);
 }
 
-- (void)testAccessibilityHintDefaultIsNotNil {
+// TODO (b/120145862) Deflake and reenable this test
```

Comment:
Tip: Next time please use a GitHub issue link instead of internal links.  I think it's easier for external contributors if we link to GitHub issues rather than internal links. Since they cannot see the `b/` content, it will be harder to find the actual work to be done later.

Reply:
Good point, will do that going forward